src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -2271,14 +2274,21 @@ private static bool CanReplaceWithReducedName(this NameSyntax name, TypeSyntax r
                 IsAmbiguousCast(name, reducedName) ||
                 IsNullableTypeInPointerExpression(name, reducedName) ||
                 name.IsNotNullableReplaceable(reducedName) ||
-                IsQualifiedNameInUsingDirective(semanticModel, name, reducedName))
+                (IsQualifiedNameInUsingDirective(semanticModel, name, reducedName) && !IsGlobalAliasQualifiedName(name)))
             {
                 return false;
             }
 
             return true;
         }
 
+        private static bool IsGlobalAliasQualifiedName(NameSyntax name)
+        {
+            return name is AliasQualifiedNameSyntax aliasName
+                ? aliasName.Alias.Identifier.IsKind(SyntaxKind.GlobalKeyword)
+                : false;
+        }
```

Comment:
1. shoudl be commented.  
2. can just be `=> name is AliasQual aliasName && ...;`